src/main/javascript/view/OperationView.js

```diff
@@ -19,6 +19,15 @@ SwaggerUi.Views.OperationView = Backbone.View.extend({
     this.parentId = this.model.parentId;
     this.nickname = this.model.nickname;
     this.model.encodedParentId = encodeURIComponent(this.parentId);
+
+    // Fix for duplicate ? in queries where the path already
+    // contains a query parameter
+    this.model.rawPath = this.model.path;
+    var tokens = this.model.rawPath.split('?');
+    if (tokens.length > 1) {
+      this.model.path = tokens[0];
+    }
```

Comment:
Ah yes, so there's also a change on the noodle-api side to add the category parameter back as a fixed value. I'll put in a PR for that one as well and link it here.